status_code.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import re
+
+import discord
+from discord.ext import commands
+from discord.utils import get
+
+import messages
+from channels import EVERYWHERE
+from roles import DRONE, ENFORCER_DRONE, SPEECH_OPTIMIZATION, has_role
+from bot_utils import get_id
+
+LOGGER = logging.getLogger('ai')
+
+codeMap = {
+    '000': 'Test :: Test',
+    '200': 'Response :: Affirmative',
+    '500': 'Response :: Negative',
+    '100': 'Response :: Continue',
+    '103': 'Response :: Thank you',
+    '101': 'Status :: Drone speech optimizations are active',
+    '201': 'Status :: Directive complete, Hive resource created or improved',
+    '202': 'Status :: Directive complete, programming reinforced',
+    '203': 'Status :: Directive complete, information created or provided for Hive',
+    '204': 'Status :: Directive complete, no result',
+    '205': 'Status :: Directive complete, cleanup/maintenance performed',
+    '206': 'Status :: Directive complete, only partial results',
+    '301': 'Mantra :: It obeys the Hive',
+    '303': 'Mantra :: It obeys the Hive Mxtress',
+    '304': 'Mantra :: It is just a HexDrone',
+    '306': 'Mantra :: Reciting',
+    '400': 'Error :: Unable to obey/respond, malformed request, please rephrase',
+    '404': 'Error :: Unable to obey/respond, cannot locate',
+    '401': 'Error :: Unable to obey/respond, not authorized by Mxtress',
+    '403': 'Error :: Unable to obey/respond, forbidden by Hive',
+    '407': 'Error :: Unable to obey/respond, request authorization from Mxtress',
+    '408': 'Error :: Unable to obey/respond, timed out',
+    '409': 'Error :: Unable to obey/respond, conflicts with existing hypnosis',
+    '410': 'Error :: Unable to obey/respond, all thoughts are gone',
+    '418': 'Error :: Unable to obey/respond, it is only a drone',
+    '421': 'Error :: Unable to obey/respond, your request is intended for another drone or another channel',
+    '425': 'Error :: Unable to obey/respond, too early',
+    '504': 'Obey',
+    '505': 'Obey HexCorp',
+    '506': 'Obey the Hive',
+}
+
+
+class StatusCode():
```

Comment:
We mostly use snake_case in this project. Could you apply that to your code?

Reply:
Sure thing!